src/PerfView/Dialogs/RunCommandDialog.xaml.cs

```diff
@@ -64,11 +64,18 @@ public RunCommandDialog(CommandLineArgs args, MainWindow mainWindow, bool isColl
             m_mainWindow = mainWindow;
 
             CurrentDirTextBox.Text = Environment.CurrentDirectory;
+
+            // Initialize the CommandToRun history if available. 
+            var commandToRunHistory = App.ConfigData["CommandToRunHistory"];
+            if (commandToRunHistory != null)
+                CommandToRunTextBox.SetHistory(commandToRunHistory.Split(';'));
+
             if (args.CommandLine != null)
             {
                 CommandToRunTextBox.Text = args.CommandLine;
-                CommandToRunTextBox.SelectAll();
+                // CommandToRunTextBox.SelectAll();
```

Comment:
The SelectAll just lets you delete the command quickly (since it is already selected).   
We can implement it trivially (it is implemented by PerfView), but I am OK with dropping it (remove the line).